docs/news.rst

```diff
@@ -3,6 +3,108 @@
 Release notes
 =============
 
+.. _release-2.9.0:
+
+Scrapy 2.9.0 (YYYY-MM-DD)
+-------------------------
+
+Highlights:
+
+-   Per-domain request settings.
```

Comment:
I wonder if we should call them something other than “request settings”, as that could be misleading (e.g. can I now automatically set a meta key-value pair for requests targeting a specific domain?).

What about “download settings” (since the setting is `DOWNLOAD_SLOTS`).